CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
+cmake_minimum_required (VERSION 3.0)
```

Comment:
@ignatenkobrain RHEL/CentOS 7.2 has CMake 2.8.11 by default. EL7.3 will have CMake 2.8.12. CMake 3.5 is available in EPEL from the the `cmake3` package.